contrib/pkg/createcluster/create.go

```diff
@@ -375,6 +380,21 @@ func (o *Options) Validate(cmd *cobra.Command) error {
 		}
 	}
 
+	if o.BoundServiceAccountSigningKeyFile != "" {
+		if o.Cloud != cloudAWS {
```

Comment:
anyone should be allowed to bring a custom signing key.

Reply:
probably doesn't require manual mode below either does it?